xcell/cls/data.py

```diff
@@ -23,31 +24,48 @@ def __init__(self, data_path='', data={}, override=False):
                              None set.')
 
         os.makedirs(self.data['output'], exist_ok=True)
-        self._check_yml_in_outdir(override)
+        self._check_yml_in_outdir(override, ignore_existing_yml)
         self.tr_matrix = None
         self.cl_tracers = {'wsp': None, 'no_wsp': None}
         self.cov_tracers = {'wsp': None, 'no_wsp': None}
 
     def get_bias(self, name):
         return self.data['tracers'][name].get('bias', 1.)
 
-    def _check_yml_in_outdir(self, override=False):
+    def _check_yml_in_outdir(self, override=False, ignore_existing_yml=False):
         outdir = self.data['output']
         fname = os.path.join(outdir, '*.yml')
         files = glob(fname)
-        if (len(files) == 1) and (not override):
+        # Start if-else with sanity checks
+        if override and ignore_existing_yml:
+            raise ValueError('Only one of override or ignore_existing_yml can '
+                             + 'be set.')
+        elif len(files) > 1:
+            raise ValueError(f'More than 1 YML file in outdir: {outdir}.')
+        elif override:
```

Comment:
I'm not sure what the difference between `override` and `ignore_existing_yml` are. I mean I can see they do different things, but I don't understand why you need both of them

Reply:
It might also be nice (potentially as part of this PR) to make the code crash if there's already a yaml file that is not the same as the input one. This is what cobaya does (although comparing yaml files may not be trivial). We can discuss offline.